### DISCUSSION TITLE
Subdirectory for Servers?

### DISCUSSION THREAD

**Discussion by andyewing** [netbox-community/devicetype-library#772] [Ideas]
I end up dealing with a lot of Dell and IBM servers, but it seems like adding device-types for those would clutter the existing folders for those vendors. Would it make sense to add a 'server-types' folder in the repo for servers/appliances?

**Comment by jeremystretch** [comment]
Personally I'd rather keep it flat, with _all_ models listed directly under the manufacturer. Eventually I'd love to be able to integrate NetBox directly with the library and I worry that adding an arbitrary layer of organization might complicate that effort.

**Comment by danner26** [comment]
@andyewing If Jeremy's answer is sufficient, please mark it as answered. If you have anymore questions feel free to let us know! Thank you.

**Reply by andyewing** [reply]
Strange. I don't see the check mark option on Jeremy's comment. I'd mark it as answered if I could.

**Reply by jeremystretch** [reply]
This post was made in "ideas;" GitHub allows you to designate an answer for posts only under the "Q&A" forum.

**Reply by danner26** [reply]
That's the part I was missing, thanks Jeremy

**Comment by andyewing** [comment]
Makes sense!
